.github/workflows/python-app.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       uses: conda-incubator/setup-miniconda@v3
       with:
         auto-activate-base: false
-        python-version: 3.8
+        python-version: "3.10"
```

Comment:
@Vittorio-Caggiano - check this.

Reply:
VC: Moved down to 3.8 to better capture the minimum version of python supported.
In principle this should be on all python versions we plan to support